lib/DI/ContainerConfigurator.php

```diff
@@ -57,6 +57,7 @@ function configure(ContainerBuilder $container) {
     // Config
     $container->autowire(\MailPoet\Config\AccessControl::class)->setPublic(true);
     $container->autowire(\MailPoet\Config\Activator::class)->setPublic(true);
+    $container->autowire(\MailPoet\Config\Populator::class)->setPublic(true);
```

Comment:
Does this have to be public? I don't think so...

Reply:
I agree, it does not have to be public.